tests/components/plex/test_init.py

```diff
@@ -293,10 +290,33 @@ def __init__(self):
 
     new_entry = MockConfigEntry(domain=const.DOMAIN, data=DEFAULT_DATA)
 
+    # Test with account failure
     with patch(
         "plexapi.server.PlexServer", side_effect=WrongCertHostnameException
-    ), patch("plexapi.myplex.MyPlexAccount", return_value=MockPlexAccount()):
+    ), patch(
+        "plexapi.myplex.MyPlexAccount", side_effect=plexapi.exceptions.Unauthorized
+    ):
         old_entry.add_to_hass(hass)
+        assert await hass.config_entries.async_setup(old_entry.entry_id) is False
+        await hass.async_block_till_done()
+
+    assert old_entry.state == ENTRY_STATE_SETUP_ERROR
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)